saleor/core/weight.py

```diff
@@ -35,13 +35,26 @@ def zero_weight():
     return Weight(kg=0)
 
 
-def convert_weight(weight, unit):
+def convert_weight(weight: Weight, unit: str) -> Weight:
```

Comment:
Maby better to use `WeightUnits` instead of `str` as `unit` type?

Reply:
Would be nice if we could use enum here.

Reply:
Also let's add a docstring here, even very simple that it just coverts weight to given unit.

Reply:
There is a short description in a comment. I can covert it to the docstring

Reply:
Must be `str` because `getattr` require `str`.